HDF5Examples/README.md

```diff
@@ -48,17 +48,17 @@ HDF5 SNAPSHOTS, PREVIOUS RELEASES AND SOURCE CODE
 --------------------------------------------
 Full Documentation and Programming Resources for this HDF5 can be found at
 
-   https://portal.hdfgroup.org/documentation/index.html
+   https://support.hdfgroup.org/documentation/HDF5/index.html
 
 Periodically development code snapshots are provided at the following URL:
-    
-   https://gamma.hdfgroup.org/ftp/pub/outgoing/hdf5/snapshots/
+
+   https://github.com/HDFGroup/hdf5/releases
```

Comment:
https://github.com/HDFGroup/hdf5/releases/tag/snapshot

Reply:
releases is the more general location - snapshot is just develop, snapshot-1.14 is for the releasable branch

Reply:
The snapshot URL is probably better as it is more closely related to the preceding sentence "Periodically development code snapshots are provided at the following URL:", and there are no develop code "releases".  However, the last snapshot listed for develop is dated May 25.  Should there be monthly/bi-weekly/weekly develop and 1.14 snapshots deployed, maybe with a reduced number of assets?